docs/module/lag.md

```diff
@@ -9,7 +9,7 @@ LAG is currently supported on these platforms:
 | Operating system      | LACP | Static | Passive<br>LACP |
 | --------------------- |:--:|:--:|:--:|
 | Arista EOS [❗](caveats-eos) | ✅ | ✅ | ✅ |
-| Cumulus Linux         | ✅ | ✅ | ❌  |
+| Cumulus Linux 4.x/5.x | ✅ | ✅ | ❌  |
```

Comment:
As we have two distinct devices and completely different configuration methods (and quite often different functionality), it would be better to keep Cumulus Linux 4.x and Cumulus Linux NVUE apart (and yes, we should be consistent, any suggestions?)

Reply:
Copied the entry from vlan module

Reply:
I'm not saying that one is correct 🤷‍♂️